satellite-overpass-identification-tool/satellite_overpass_identification_tool.py

```diff
@@ -96,7 +97,16 @@ def get_passtimes(start_date, end_date, csvoutpath, lat, lon, SPACEUSER, SPACEPS
 # Write CSV of all pass information.
 def csvwrite(startdate, enddate, lat, lon, rows, outpath):
     fields = ["Date", "Aqua pass time", "Terra pass time"]
-    filename = f"{outpath}/passtimes_lat{lat}_lon{lon}_{''.join(startdate)}_{''.join(enddate)}.csv"
+    outpath_ = pathlib.Path(outpath)
+    
+    if outpath_.is_dir():
+        filename = outpath_ / pathlib.Path(f"passtimes_lat{lat}_lon{lon}_{''.join(startdate)}_{''.join(enddate)}.csv")
```

Comment:
This line is a bit too busy and long. Care to split it up into two or more lines?

Reply:
Sure, how about this?